addons/packages/vsphere-cpi/1.23.0-alpha.1/README.md

```diff
@@ -42,7 +44,7 @@ None
 |`vsphereCPI.nsxt.rootCAData` | Optional | The certificate authority for the server certificate for locally signed certificates. Default: `""`. |
 | `vsphereCPI.nsxt.secretName` | Optional | The name of secret that stores CPI configuration. Default: `cloud-provider-vsphere-nsxt-credentials`. |
 | `vsphereCPI.nsxt.secretNamespace`| Optional | The namespace of secret that stores CPI configuration. Default: `True`. |
-| `vsphereCPI.clusterAPIVersion`| Optional for `vsphereParavirtual` | Used in `vsphereParavirtual` mode, defines the Cluster API versions. Default: `run.tanzu.vmware.com/v1alpha2` |
+| `vsphereCPI.clusterAPIVersion`| Optional for `vsphereParavirtual` | Used in `vsphereParavirtual` mode, defines the Cluster API versions. Default: `run.tanzu.vmware.com/v1beta1` |
```

Comment:
Could you also update the group version to align with https://github.com/kubernetes-sigs/cluster-api/blob/main/api/v1beta1/groupversion_info.go#L29